webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/AccessTokenService.scala

```diff
@@ -102,6 +95,8 @@ trait AccessTokenService {
       case UserAccessAnswer(true, _) =>
         block
       case UserAccessAnswer(false, Some(msg)) =>
+        // Note that this should be kept in sync with DSLegacyApiController.withResolvedDatasetId
+        // to make the errors indistinguishable.
```

Comment:
Then maybe lets make it a message in conf/messages?

Reply:
Hmm, I’m not sure how to achieve that, because the text in DSLegacyApiController.withResolvedDatasetId already includes the msg that is appended here. Open for ideas.

But tbh I just hope we can ditch these old API versions somewhat soon, which would make this problem just go away.